tests/acceptance/30_generic_methods/file_copy_from_local_source_recursion.cf

```diff
@@ -52,9 +52,10 @@ bundle agent test
 bundle agent check
 {
   vars:
-    "permissions_test_mode"  string => "/usr/bin/test $(/usr/bin/stat -c %a ${init.destination_file}) = \"${init.mode}\"";
-    "permissions_test_owner" string => "/usr/bin/test $(/usr/bin/stat -c %U ${init.destination_file}) = \"${init.owner}\"";
-    "permissions_test_group" string => "/usr/bin/test $(/usr/bin/stat -c %G ${init.destination_file}) = \"${init.group}\"";
+    "owner_id"                  int => getuid("${init.owner}");
```

Comment:
Because getuid fails if you pass it a number :-( 

I've tried to always use a bane for the user, but for the group I use 0 because the "root" group name is different across OSes